src/components/Nav/index.tsx

```diff
@@ -97,25 +96,6 @@ const HomeLogo = styled(HomeIcon)`
   }
 `
 
-const Span = styled.span`
-  padding-left: 0.5rem;
-`
-
-const ThemeToggle = styled(NakedButton)`
-  margin-left: 1rem;
-  display: flex;
-  align-items: center;
-  &:hover {
-    svg {
-      fill: ${(props) => props.theme.colors.primary};
-    }
-  }
-`
-
-const NavIcon = styled(Icon)`
-  fill: ${(props) => props.theme.colors.text};
-`
-
```

Comment:
Wondering why we have removed these, were they unused components?

Reply:
If so, thanks :)

Reply:
Yes they were all 3 unused :smile: